### PR TITLE
Feat base payload creation

### PR DESCRIPTION
**Pull request by ludovic-str**

### CODE REVIEW

File: common/packet/DestroyEntity.hpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+namespace RType::Packet {
+    /**
+     * @brief DestroyEntity is a class that represents a destroy entity packet
+     */
+    struct DestroyEntity {
+        /**
+         * @brief Constructs a new DestroyEntity object
+         * @param id The id of the entity
+         */
+        DestroyEntity(std::string id = 0) {
```
Comment (Tomeiru) [inline common/packet/DestroyEntity.hpp:14]: When you init a std::string dont use 0 but ""

File: common/packet/EntityPosition.hpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+#include <string>
+
+namespace RType::Packet {
+    /**
+     * @brief EntityPosition is a class that represents an enemy position packet
+     */
+    struct EntityPosition {
+
+        EntityPosition(std::string id = 0, float x = 0, float y = 0) : _x(x), _y(y) {
```
Comment (Tomeiru) [inline common/packet/EntityPosition.hpp:11]: When you init a std::string dont use 0 but ""

**Review by Tomeiru — CHANGES_REQUESTED** [review]

File: common/packet/DestroyEntity.hpp (hunk 1 above)
Comment (ludovic-str) [reply common/packet/DestroyEntity.hpp]: https://github.com/Tomeiru/R-Type/pull/12/commits/5ff6c077dd33d7e03bd4fafc276448c5e5765619

File: common/packet/EntityPosition.hpp (hunk 2 above)
Comment (ludovic-str) [reply common/packet/EntityPosition.hpp]: https://github.com/Tomeiru/R-Type/pull/12/commits/5ff6c077dd33d7e03bd4fafc276448c5e5765619

**Review by Tomeiru — APPROVED** [review]
LGTM

**Pull request merged**